### PR TITLE
add sdn live migration on 4.15 version

### PR DESCRIPTION
**Pull request by zhaozhanqi**
As sdn live migration is supported on 4.15 version https://issues.redhat.com/browse/NP-1063

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-sdn-live-migration-ovn-fips-f28

**Comment by zhaozhanqi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-sdn-live-migration-ovn-fips-f28

**Comment by zhaozhanqi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-sdn-live-migration-ovn-fips-f28

**Comment by zhaozhanqi** [comment]
this test [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/56542/rehearse-56542-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-sdn-live-migration-ovn-fips-f28/1833822212558163968) passed 

@liangxia Could you please take a look and help approve this?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml [hunk 1]
```diff
@@ -2600,6 +2622,18 @@ tests:
...
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy-minimal-permission
+- as: nutanix-ipi-sdn-live-migration-ovn-f28
+  cluster: build01
+  cron: 6 19 6 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2632]: We have FIPS_ENABLED here, but not indicated in the test name.  So I'm not sure whether you want to test fips or not.

Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2632]: ah, thanks for catching it. 
I'd like to enable it even though it's not important for migration.  Update the profile name with `fips`, thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 2]
```diff
@@ -1159,6 +1159,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: azure-ipi-sdn-live-migration-ovn-f28
+  cron: 33 17 4 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
```
[the remark this answers is not in the document]
Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml]: @liangxia updated all, could you help check again? thanks

**Comment by zhaozhanqi** [comment]
@liangxia FYI, no changes compare with last time push. just resolve the conflicts.

**Pull request merged**